devel/libert_util/src/stepwise.c

```diff
@@ -457,11 +465,11 @@ void stepwise_free( stepwise_type * stepwise ) {
 
 }
 
-void stepwise_printf( const stepwise_type * stepwise , const int index ) {
+void stepwise_printf( const stepwise_type * stepwise , const char* key, const int active_index, const int global_index ) {
```

Comment:
Seems this has now aquired `fwd_step_enkf` specific arguments? I would prefer if:

1. The print functionality was moved to `fwd_step_log`.
2. The `stepwise_printf` function was just removed.